charts/routing-actions/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: routing-actions
-version: 0.1.5
+version: 0.18.0
```

Comment:
This version iv `0.18.0` but I see the version in the controller is `0.18.1`. Is that intentional?

Reply:
There is no reason to keep them in sync at the patch level. This is the first update to the routing-action chart. The controller chart was already at 18.0